components/tensorboard-controller/api/v1alpha1/tensorboard_types.go

```diff
@@ -0,0 +1,60 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// TensorboardSpec defines the desired state of Tensorboard
+type TensorboardSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+	LogsCloudPath string `json:"logscloudpath"`
+}
+
+// TensorboardStatus defines the observed state of Tensorboard
+type TensorboardStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
```

Comment:
Are you asking to add deployment status in reconcile ? I'm hesitated to do so because:
1. It's simple to for customer get deployment status by running ```kubectl get```
2. It'll take a while for deployment to spin up, in reconcile we most likely to get pending status.

Reply:
per offline chat with @kunmingg , added to surface status.